### ISSUE TITLE
TTL output baseURI incorrect

### ISSUE THREAD

**Issue by ronaldtse** [geolexica/geolexica-server#54]
From @ghobona (thank you!)

> please check whether the base URI is correct on the definitions.
> https://isotc211.geolexica.org/api/concepts/880.ttl
>
> I think the baseURI value should conform with the IETF standard for URIs https://tools.ietf.org/html/rfc3986
>
> See clause 3. A URI is supposed to have a scheme that is separated by a “:” from the other component parts. Typical examples include http, https, urn, and so on.

Right now the TTL output starts with:

```ttl
# baseURI: "/concepts/880/"
# imports: http://purl.org/dc/terms/
# imports: https://www.geolexica.org/api/rdf-profile
# imports: http://www.w3.org/2004/02/skos/core

@prefix : </concepts/880/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
...
```

I believe `baseURI` and `@prefix` should both have the full `https://...` hostname.

**Comment by skalee** [comment]
Yes, I'm aware of that. I was going to fix it when we finally switch to RDF gem, same applies to other improvements described in #2.

**Comment by ronaldtse** [comment]
@skalee Will this be fixed soon? Thanks.
